### PR TITLE
Env update to support building with ALE

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

Tested working locally on CentOS, MacOS and Ubuntu. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -4,14 +4,14 @@ channels:
   - defaults
 
 dependencies:
-  - ale>=0.3.2
+  - ale=0.5.0
```
Comment (jessemapel) [inline environment.yml]: Can we make this a greater than or does it need to be 0.5.0?

Comment (Kelvinrr) [reply environment.yml]: 0.6 introduced an API breaking change and the version of ALE that was used for ISIS is based on 0.5.0

**Comment by Kelvinrr** [comment]
Fun fact: you need to have a Jenkinsfile change for Jenkins to recognize the PR. Seems Jenkins only scans diffs when looking for Jenkinsfiles and not the merge PR with the target branch.

**Comment by jessemapel** [comment]
It looks like several of the errors we're getting are from switching from ISIS specific builds to conda-forge builds. For example, the conda-forge bullet build uses floats instead of doubles which causes seg-faults.

Here's the packages from us used in the 3.9.0 release:
```
bullet                    2.86.1                        0    usgs-astrogeology
flann                     1.8.4                         0    usgs-astrogeology
geos                      3.5.1                h6bb024c_1    usgs-astrogeology
gmm                       5.0                           0    usgs-astrogeology
isis3                     3.9.0                    py36_0    usgs-astrogeology
jama                      125                           0    usgs-astrogeology
kakadu                    1                             0    usgs-astrogeology
libprotobuf               3.5.2                h6bb024c_0    usgs-astrogeology
patchelf                  0.9                           0    usgs-astrogeology
pcl                       1.8.1                h6bb024c_1    usgs-astrogeology
protobuf                  3.5.2            py36h6bb024c_0    usgs-astrogeology
qhull                     7.2.0                         0    usgs-astrogeology
qt                        5.9.6                         0    usgs-astrogeology
qwt                       6.1.3                         0    usgs-astrogeology
tbb                       4.4                           0    usgs-astrogeology
tnt                       126                           0    usgs-astrogeology
xalan-c                   1.11                          0    usgs-astrogeology
xerces-c                  3.1.4                         0    usgs-astrogeology
```

Here's the packages from us when I make a new environment off this:
```
geos                      3.5.1                h6bb024c_1    usgs-astrogeology
gmm                       5.0                           0    usgs-astrogeology
jama                      125                           0    usgs-astrogeology
kakadu                    1                             0    usgs-astrogeology
qwt                       6.1.3                         0    usgs-astrogeology
tnt                       126                           0    usgs-astrogeology
xalan-c                   1.11                          0    usgs-astrogeology
```

I think we need to pin some of these back so that we get the special builds provided by us.

**Review by scsides — APPROVED** [review]
The decision has been made to merge this and work on centOS after the other OSs are working.

**Pull request merged**